wbooks/src/App/Main/BookDetail/Details/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import './styles.css';
+
+class Details extends React.Component {
+  render() {
+    let cover = (
```

Comment:
You don't need a variable here.
You can move the conditional to the return bewtween `{}`.
```js
<div className="section book-details">
  {
    this.props.bookInfo.image_url ?
      <img className="book-cover" alt="" src={this.props.bookInfo.image_url} /> :
      <div className="book-cover no-cover"></div>
  }
  ...
</div>
```
(also if you don't reassign the variable use `const`)